docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingSample.cs

```diff
@@ -10,71 +13,88 @@ public static void Example()
             var mlContext = new MLContext();
 
             // Get a small dataset as an IEnumerable and convert it to an IDataView.
-            var data = SamplesUtils.DatasetUtils.GetInfertData();
-            var trainData = mlContext.Data.LoadFromEnumerable(data);
+            var rawData = GetData();
+
+            // Printing the input data. 
+            Console.WriteLine("Age\t Salary");
+            foreach (var row in rawData)
+                Console.WriteLine($"{row.Age}\t {row.Salary}");
+            // Expected output:
+            // Age      Salary
+            // 26       40000
+            // 35       80000
+            // 34       10000
+            // 28       100000
```

Comment:
if you define the data inline, you don't need this preview. it's already obvious from code.